includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -33,7 +33,8 @@ public function getData() {
 				'title' => [
 					'type' => 'text',
 					'value' => 'Fandom powered by Wikia'
-				]
+				],
+				'tracking_label' => 'logo',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "tracking_label" 30 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)